src/core/main.c

```diff
@@ -647,6 +648,8 @@ static int config_parse_join_controllers(const char *unit,
 
         return 0;
 }
+int config_parse_ctrl_alt_del_burst_action(const char *unit, const char *filename, unsigned line, const char *section, unsigned section_line, const char *lvalue, int ltype, const char *rvalue, void *data, void *userdata);
+DEFINE_CONFIG_PARSE_ENUM(config_parse_ctrl_alt_del_burst_action, ctrl_alt_del_burst_action, CtrlAltDelBurstAction, "Failed to parse service restart specifier");
```

Comment:
Maybe call this "CADBurstAction" internally everywhere? (The config file name is OK.) Those function names are awfully long.